src/native/managed/cdac/src/mscordaccore_universal.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The src directory should be renamed to `mscordaccore_universal` so that it matches the project name

Reply:
I meant the path to be src/native/managed/cdac/mscordaccore_universal/mscordaccore_universal.csproj , but it looks like you have also deleted `cdac` from the path during the rename.

I think it would make sense for `mscordaccore_universal` to be under `cdac` directory since it is part of cdac project.

Reply:
:-) - sorry

Reply:
Fixed